test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -92,6 +93,10 @@ def __init__(self, num_ignored: int) -> None:
 
 
 class TestFSDPIgnoredModules(FSDPTest):
+    @property
+    def world_size(self):
```

Comment:
Based on https://github.com/pytorch/pytorch/pull/112406/, we need this `min()` to avoid errors when running with 1 GPU. Otherwise, we want to run with world size 2.

Reply:
could this be added to `FSDPTest` parent class if this becomes a common pattern?

Also I kinda curious when we'll run the test with only 1 GPU given that all the test cases are decorated with `@skip_if_lt_x_gpu(2)`?

Reply:
Sounds good! Wecan  add it to the `FSDPTest` class. (Though, I think many of the child test classes override `world_size` themselves, so they will need to be adjusted too).

From https://github.com/pytorch/pytorch/pull/112406, the decorator is not sufficient to correctly skip the test. IIRC, the order of operations is like: first try to initialize the process group with world size 2; then the decorator skips the test since there is an insufficient number of GPUs. However, the error happens in the 1st step.

Maybe this error in initializing the process groups is relatively new, causing us to require changing this `world_size` definition. I can revisit this when I open PRs for per-parameter FSDP.